graylog2-server/src/main/java/org/graylog/events/rest/EventDefinitionsResource.java

```diff
@@ -100,6 +102,7 @@ public EventDefinitionsResource(DBEventDefinitionService dbService,
     @Path("{definitionId}")
     @ApiOperation("Get an event definition")
     public EventDefinitionDto get(@ApiParam(name = "definitionId") @PathParam("definitionId") @NotBlank String definitionId) {
+        checkPermission(RestPermissions.EVENT_DEFINITIONS_READ);
```

Comment:
Same here, we can check for the actual instance permission:

```suggestion
        checkPermission(RestPermissions.EVENT_DEFINITIONS_READ, definitionId);
```